### PR TITLE
Hide controls when showing overlay

### PR DESCRIPTION
**Pull request by egreaves**
Hide `controls` via CSS when `sharing` or `related` overlays are showing. This circumvents the need to override the `controls` config value. JW7-1510

### CODE REVIEW

File: src/css/flags/overlay.less [hunk 1]
```diff
@@ -5,4 +5,9 @@
     .jw-controls-right .jw-logo {
         display: none;
     }
+    &-sharing, &-related {
```
Comment (pajong) [inline src/css/flags/overlay.less]: I think we only have sharing and related overlays, so we do not need to specify &-sharing, &-related

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jwplayer",
-  "version": "7.3.0-beta.1",
+  "version": "7.3.0-beta.2",
```
Comment (pajong) [inline package.json]: We don't want to commit this together.

Comment (egreaves) [inline package.json]: Agreed.

File: src/css/flags/overlay.less (hunk 1 above)
Comment (egreaves) [inline src/css/flags/overlay.less]: This is currently true. I was initially going to do it that way, but after discussing with @robwalch, I concur that it's safer to limit the behavior to just the plugins for now, allowing something else to implement overlays slightly differently if the need arises. We could alternatively make it `&-plugin`, though, and apply that class to both.

Comment (pajong) [inline src/css/flags/overlay.less]: Cool, I will merge sharing and related PRs. Just need this one to get the package.json commit out.

**Pull request merged**